### DISCUSSION TITLE
Successful Android Auto on Porsche 718 Cayman

### DISCUSSION THREAD

**Discussion by Cay718** [Mr-MIBonk/M.I.B._More-Incredible-Bash#202] [Q&A]
No issues at all enabling Android Auto on my 2017 Cayman.

Some notes

- Use a quality branded SD Card.  Do a full format FAT32 of the card in a PC, not the "Quick Format"
- A battery charger is essential to keep power to the PCM.  I used a 5 AMP charger which did the job
- The touchscreen did not work in GEM.  Turn the right knob to move the cursor up and down and push it to select an item.
- If you are of a nervous disposition (like me!), walk away from the car for few minutes while M.I.B installs
- A reboot of the PCM just shows a blank screen, it does not power up the PCM to the HOME screen.  Press the ON button to get back to the HOME screen.
- The GEM install stayed at 0% for quite a while.  Don't panic, it does complete - walk away!
- Do a backup to the SD Card as per the instructions
- The Swdlautorun.txt file gets renamed automatically to _Swdlautorun.txt so inserting the SD card does not execute the M.I.B install again (I did the IFS-ROOT patch separately from the M.I.B install)

**Comment by nassib92** [comment]
Hello, 

Thanks for the advice! 

One question, if I do not have a battery charger, can I do it while the car is running instead?

**Reply by Cay718** [reply]
The advice is not to patch with the engine running.
